core/src/main/java/org/apache/accumulo/fate/util/Retry.java

```diff
@@ -176,14 +174,14 @@ public long retriesCompleted() {
     return retriesDone;
   }
 
-  public void waitForNextAttempt() throws InterruptedException {
+  public void waitForNextAttempt(Logger log, String operationDesc) throws InterruptedException {
```

Comment:
I wonder if having the method signature as `waitForNextAttempt(Consumer<String> log, String operationDesc)` would be of use here so that way you could specify the log level. Something like: `waitForNextAttempt(log::debug, "About to retry!")` or somehow wrapping the logging of the message into a single functional variable that is passed into `waitForNextAttempt()` as its only parameter. Haven't looked into this much but just putting some ideas out in case they could lead to improvements.